app/src/docs/assets/Downloads.doc.tsx

```diff
@@ -1,104 +1,55 @@
-            link: 'https://epam.sharepoint.com/:f:/r/sites/EPAMDesignPlatform/Shared%20Documents/General/Projects/CG%20ART%20STREAM?csf=1&web=1&e=V3vkJK',
-            image: Artbord,
-            libraryType: 'figma',
-        },
-    ],
-    UUI4: [
-        {
-            title: 'UUI4 Library',
-            additionalInfo: 'Requires Sketch 70 or greater',
-            link: 'sketch://add-library?url=https%3A%2F%2Fraw.githubusercontent.com%2Fyaroslav-zonov%2FUUIDesign%2Fmain%2FUUI%25204.0%2FUUI4.xml',
-            image: Sketch,
-            libraryType: 'sketch',
-        }, {
-            title: 'UUI4 Components Library',
-            additionalInfo: 'Requires Figma 97 or greater',
-            link: 'https://www.figma.com/file/UyChXPLmyv5zMrOU37KdUL/UUI4-(Components)?node-id=280%3A85528',
-            image: Artbord,
-            libraryType: 'figma',
-        }, {
-            title: 'UUI Assets Library',
-            additionalInfo: 'Requires Figma 97 or greater',
-            link: 'https://www.figma.com/file/3mpAy3BEZ75n5GJEZ5UV8z/UUI-(Assets)?node-id=0%3A2097',
+            title: 'UUI Component Guidelines',
+            additionalInfo: 'Guidelines of UUI components',
+            link: 'https://www.figma.com/file/dYwal6PrxrPQWg1lL6ijpn/UUI-Guidelines?type=design&node-id=1886%3A625715&mode=design&t=3ve5cI7s6roXxHxw-1',
             image: Artbord,
             libraryType: 'figma',
         }, {
             title: 'UUI Patterns',
-            additionalInfo: 'Requires Figma 97 or greater',
-            link: 'https://www.figma.com/file/qb7WHgBkyBpovFlOZRe30p/UUI-Patterns',
+            additionalInfo: 'Templates guidelines',
+            link: 'https://www.figma.com/file/qb7WHgBkyBpovFlOZRe30p/UUI-Patterns?type=design&node-id=0%3A1&mode=design&t=LByFqSyb9pwJ6FMX-1',
             image: Artbord,
             libraryType: 'figma',
         }, {
             title: 'UUI Illustrations',
-            additionalInfo: 'Requires Figma 97 or greater',
-            link: 'https://epam.sharepoint.com/:f:/r/sites/EPAMDesignPlatform/Shared%20Documents/General/Projects/CG%20ART%20STREAM?csf=1&web=1&e=V3vkJK',
+            additionalInfo: 'Illustrations pack in addition (promo images, empty states, etc.)',
+            link: 'https://www.figma.com/file/fNIMFXueuk3pfJzL4lWCex/UUI-Illustrations?type=design&t=f7W2ssNZnh1LeBJK-6',
             image: Artbord,
             libraryType: 'figma',
         },
     ],
 };
 
 const assets = {
-    UUI3: {
-        fonts: 'https://static.cdn.epam.com/uploads/690afa39a93c88c4dd13758fe1d869d5/EPM-UUI/UUI3/Fonts_UUI3.7z',
-        logos: 'https://static.cdn.epam.com/uploads/690afa39a93c88c4dd13758fe1d869d5/EPM-UUI/UUI3/Logotypes_UUI3.7z',
-        icons: 'https://static.cdn.epam.com/uploads/690afa39a93c88c4dd13758fe1d869d5/EPM-UUI/UUI3/Icons_UUI3.7z',
-        illustrations: 'https://epam.sharepoint.com/:u:/s/EPAMUII3/EahAf6j0ZERKqnbgZwTPFjYBX1HJJC9n845j6xC3FL-gKg?e=K0LQU1',
-    },
-    UUI4: {
+    UUI: {
```

Comment:
Let's make it plain object without nested UUI item, because now we are displaying only UUI here